test/production/graceful-shutdown/pages/api/long-running.ts

```diff
@@ -0,0 +1,6 @@
+export const LONG_RUNNING_MS = 400
```

Comment:
I had lowered this to `100` but seemed to run into less race conditions bumping it up higher. In practice, `400ms` wouldn't really be a "long running" request but should be enough to simulate the issue in tests.